control-panel/build.gradle

```diff
@@ -48,6 +48,7 @@ jar {
             'Alfresco-Spring-Configuration': 'com.github.dynamicextensionsalfresco.controlpanel',
             'Spring-Context': ';publish-context:=false',
             'Import-Package': '*, ' +
+                    'javax.annotation;version=!, ' +
```

Comment:
The '*' should move to the end, otherwise `javax.annotation` is twice in the `Import-Package` header.